srcpkgs/MemoryLatencyTest/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'MemoryLatencyTest'
+pkgname=MemoryLatencyTest
+version=0.3.2
+revision=1
+archs="~i686*"
+build_style=meson
+short_desc="Cross-platform CPU memory latency benchmark"
+maintainer="Sam Close <sam.w.close@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/ChipsandCheese/MemoryLatencyTest"
+changelog="${homepage}/blob/main/CHANGELOG.md"
```

Comment:
don't use `${homepage}` variables here, just put the full url in `changelog` and `distfiles` so they are easier to copy and paste.